raven/src/main/java/org/opencadc/raven/ArtifactAction.java

```diff
@@ -185,6 +188,13 @@ protected ArtifactAction() {
       
         // get the storage site rules
         this.siteRules = RavenInitAction.getStorageSiteRules(props);
+        String cnf = props.getFirstPropertyValue(RavenInitAction.ARTIFACT_NOT_FOUND_DEFAULT_KEY);
```

Comment:
Just a hbeads up: there should never be case-conversion when reading config; that will introduce inconsistency and confusion. However, don't fix this yet as I haven't decided on the configuration concept that matches this feature yet but it will probably change.